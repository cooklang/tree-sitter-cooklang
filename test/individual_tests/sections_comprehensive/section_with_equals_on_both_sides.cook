== Filling ==

Prepare the filling ingredients.