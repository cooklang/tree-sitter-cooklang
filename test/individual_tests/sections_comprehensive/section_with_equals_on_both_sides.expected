(recipe
  (section
    header: (section_header
      name: )
    content: (step
      (plain_text))))